sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -108,8 +108,10 @@
     "@azure/core-http": "^2.2.4",
```

Comment:
unrelated to this PR but it is weird that it depends on both core-client and core-http. Is there anything of value being imported from core-http?

Reply:
Exporter allows TokenCredential to be passed as config, that one is part of @azure/core-http

Reply:
[TokenCredential is exported by core-auth](https://github.com/Azure/azure-sdk-for-js/blob/ff0ed7ca60a20a99224a73fc677a4e1434620abb/sdk/core/core-auth/src/tokenCredential.ts#L10), could you import it from there instead?